packages/e2e-tests/fixtures/blocks/core__media-text__image-fill-no-focal-point-selected.html

```diff
@@ -0,0 +1,13 @@
+<!-- wp:media-text {"mediaId":11,"mediaType":"image","imageFill":true} -->
```

Comment:
As of today, we are using, the fixtures as part of the transforms tests to media blocks. This means we can not use URLs of images that trigger 404 errors or media ids that don't exist in the test server. The easiest solution is using a data URL. We also need to update the transforms tests structure and specify this fixture can be transformed into an image. In PR https://github.com/WordPress/gutenberg/pull/14790 we apply all the required actions to other fixtures.